src/elements/content-sidebar/versions/VersionsItemActions.js

```diff
@@ -101,13 +101,13 @@ const VersionsItemActions = ({
 
                 {showDownload && (
                     <VersionsItemAction action="download" fileId={fileId} isCurrent={isCurrent} onClick={onDownload}>
-                        <IconDownload {...ICON_SIZE} />
+                        <IconDownload height={14} width={14} />
                         <FormattedMessage {...messages.versionActionDownload} />
                     </VersionsItemAction>
                 )}
                 {showPromote && (
                     <VersionsItemAction action="promote" fileId={fileId} isCurrent={isCurrent} onClick={onPromote}>
-                        <IconUpload {...ICON_SIZE} />
+                        <IconUpload height={14} width={14} />
```

Comment:
Likewise, these all are in the drop down

Reply:
changing to 16x16